forge/src/main/java/com/gregtechceu/gtceu/common/item/tool/forge/ToolLootModifier.java

```diff
@@ -27,7 +27,7 @@ private ToolLootModifier() {/**/}
             Vec3 pos = context.getParam(LootContextParams.ORIGIN);
             BlockPos blockPos = new BlockPos(Mth.floor(pos.x), Mth.floor(pos.y), Mth.floor(pos.z));
             ItemStack tool = context.getParam(LootContextParams.TOOL);
-            boolean isSilktouch = EnchantmentHelper.getItemEnchantmentLevel(Enchantments.SILK_TOUCH, tool) > 1;
+            boolean isSilktouch = EnchantmentHelper.getItemEnchantmentLevel(Enchantments.SILK_TOUCH, tool) > 0;
```

Comment:
oh, uh, oops.

Reply:
that was totally also loom's fault, not mine